src/main/java/com/stripe/model/EventDataObjectDeserializer.java

```diff
@@ -185,13 +182,14 @@ public StripeObject deserializeUnsafeWith(CompatibilityTransformer transformer)
         transformer.transform(rawJsonObject.deepCopy(), apiVersion, eventType));
   }
 
+  @SuppressWarnings("StringSplitter")
```

Comment:
Should we use substring and avoid the warning?

Reply:
Good call. Done.